rotkehlchen/constants/location_details.py

```diff
@@ -59,5 +64,12 @@
     Location.WOO: {'image': 'woo.svg'},
 }
 for key, value in LOCATION_DETAILS.items():
-    if key in SUPPORTED_EXCHANGES:
-        value['is_exchange_with_key'] = True
+    if key in ALL_SUPPORTED_EXCHANGES:
```

Comment:
this will include the dead exchanges. Maybe you want to mark somehow if the exchange is dead?

Reply:
I need both dead exchanges, and exchanges that can be added with key

the dead exchanges, I need it in purge data section

Reply:
what I mean is that before you didn't have them and now you don't know that they are dead because there is no key telling you it unless I'm missing something

Reply:
I mark all exchanges as `is_exchange = True`,
and the ones that are still alive I mark is as `is_exchange_with_key  = True`.

It should be enough to separate them, no?

Reply:
Ah okey I understood now. Yes should be fien